### DISCUSSION TITLE
Running Docker File on macOS, linux and windows

### DISCUSSION THREAD

**Discussion by Hemanth0324** [DrAlzahraniProjects/csusb_fall2024_cse6550_team4#2] [Announcements]
Hi Team,

If you have Windows, linux or macOS, please test the latest Docker file as soon as possible.

**Comment by SAditya0410** [comment]
Hi, 

I have tested it in windows environment. It is working. PRB screenshot
![hello team 4](https://github.com/user-attachments/assets/c309ac86-7cf6-4cb2-811d-42f1436243a3)

**Comment by lakshita470** [comment]
Yes, its working on windows environment.
![Screenshot (227)](https://github.com/user-attachments/assets/2545e4d6-95ba-4d79-9943-2734b7911931)

**Comment by venkat-coyote** [comment]
After Correcting the violations. The app is running fine in my computer
OS : Windows 11
Build : Successful
Running on localhost 5004 : Successful
![Screenshot 2024-09-13 204859](https://github.com/user-attachments/assets/dab2e3de-232a-428d-96ff-b0234930fab9)
